pyanaconda/ui/tui/spokes/user.py

```diff
@@ -130,11 +126,31 @@ def status(self):
         else:
             return _("User %s will be created") % self.data.user.userList[0].name
 
+    def input(self, args, key):
+        self.dialog.wrong_input_message = None
+        try:
+            field = self.visible_fields[int(key)-1]
+        except (ValueError, IndexError):
+            pass
+        else:
+            if field.attribute == "gecos":
+                self.dialog.wrong_input_message = _("Full name can't contain ':' character")
```

Comment:
Just a simple suggestion: 

"Full name can't contain **the** ':' character"

That IMHO sounds better. :)